.github/instructions/dsc-community-style-guidelines-pester.instructions.md

```
@@ -0,0 +1,63 @@
+---
+description: Guidelines for writing and maintaining tests using Pester.
+applyTo: "**/*.[Tt]ests.ps1"
+---
+
+# Tests Guidelines
+
+## Core Requirements
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)